test/test_components.py

```diff
@@ -697,7 +697,6 @@ def test_component_functions(self):
         with pytest.raises(ValueError):
             gr.Image(type="unknown")
         image_input.shape = (30, 10)
-        assert image_input._segment_by_slic(img) is not None
```

Comment:
This was failing for me locally. I spent some time investigating but couldn't figure out the reason (it relates to image modes). But since we are planning on deprecating interpretation anyways, I think its okay to remove.